src/iop/colorequal.c

```diff
@@ -263,6 +264,24 @@ typedef struct dt_iop_colorequal_gui_data_t
   float points[NODES+1][2];
 } dt_iop_colorequal_gui_data_t;
 
+void tiling_callback(
+        struct dt_iop_module_t *self,
+        struct dt_dev_pixelpipe_iop_t *piece,
+        const dt_iop_roi_t *roi_in,
+        const dt_iop_roi_t *roi_out,
+        struct dt_develop_tiling_t *tiling)
+{
+  dt_iop_colorequal_data_t *data = (dt_iop_colorequal_data_t *)piece->data;
+
+  tiling->maxbuf = 1.0f;
+  tiling->xalign = 1;
+  tiling->yalign = 1;
+  tiling->overhead = (2 * SATSIZE + 4 * LUT_ELEM) * sizeof(float);
+  tiling->factor = 2.0f + (data->use_filter ? 3.0f : 1.0f); // include internal buffers plus gaussian
```

Comment:
By my count, I get a total of 4.25 plus gaussian (another 1.0?) for the internal buffers when filtering.

Definitely one of the more memory-intensive modules, so having tiling is important.

Reply:
also depends on scaling which is dependent on radius ...

Reply:
I was counting worst-case, since the tiling_callback is not adjusting for scaling.